pkg/scheduler/eventhandlers.go

```diff
@@ -204,7 +204,19 @@ func (sched *Scheduler) addPodToCache(obj interface{}) {
 		logger.Error(err, "Scheduler cache AddPod failed", "pod", klog.KObj(pod))
 	}
 
-	sched.SchedulingQueue.AssignedPodAdded(logger, pod)
+	// SchedulingQueue.AssignedPodAdded internally pre-filters Pods to move to activeQ while taking only in-tree plugins into consideration.
```

Comment:
I prefer the current implementation because after `SchedulerQueueingHints` feature flag goes to GA, AssignedPodAdded is no longer needed and we'll use MoveAllToActiveOrBackoffQueue only. What do you think?